tekton/images/test-runner/Dockerfile

```diff
@@ -12,9 +12,162 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM gcr.io/k8s-testimages/kubekins-e2e:v20190729-351ea95-master
+FROM debian:buster
 LABEL maintainer "Tekton Authors <tekton-dev@googlegroups.com>"
 
+ENV DEBIAN_FRONTEND noninteractive
+
+WORKDIR /workspace
+RUN mkdir -p /workspace
+ENV WORKSPACE=/workspace \
+    TERM=xterm
+
+# common util tools
+# https://github.com/GoogleCloudPlatform/gsutil/issues/446 for python-openssl
+RUN apt update && apt install -y --no-install-recommends \
+    build-essential \
+    ca-certificates \
+    curl \
+    file \
+    git \
+    jq \
+    mercurial \
```

Comment:
???

Reply:
`kubeins-e2e` and `bootstrap` from k8s.io/test-infra :sweat_smile: 
I'll clean some stuff later on :wink: